### PR TITLE
[IOTDB-1807] Compatibility of Apache IoTDB with InfluxDB - Parse InfluxQL

### PR DESCRIPTION
**Pull request by jun0315**
see https://issues.apache.org/jira/browse/IOTDB-1807

### CODE REVIEW

**Comment by jun0315** [comment]
@LIU-WEI-git  Thank you very much for your great contribution: let the expression and operator of influxdb extends the relevant classes of iotdb.
PLAT :D @SteveYurongSu

File: server/src/main/java/org/apache/iotdb/db/protocol/influxdb/constant/InfluxSQLConstant.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  public static final String SPREAD = "spread";
+  public static final String STDDEV = "stddev";
+  private static final Set<String> NATIVE_FUNCTION_NAMES =
+      new HashSet<>(
+          Arrays.asList(MIN, MAX, FIRST, LAST, MEAN, COUNT, MEDIAN, MODE, SPREAD, STDDEV, SUM));
+  private static final Set<String> NATIVE_SELECTOR_FUNCTION_NAMES =
+      new HashSet<>(Arrays.asList(MIN, MAX, FIRST, LAST));
+
+  public InfluxSQLConstant() {
+    // forbidding instantiation
```
Comment (SteveYurongSu) [inline server/src/main/java/org/apache/iotdb/db/protocol/influxdb/constant/InfluxSQLConstant.java:46]: throw exception here

File: server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
 @SuppressWarnings("unused") // some fields are for future features
 public class SQLConstant {
 
-  private SQLConstant() {
+  public SQLConstant() {
     // forbidding instantiation
```
Comment (SteveYurongSu) [inline server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java:34]: throw exception here

**Review by SteveYurongSu — COMMENTED** [review]
Overall great!

**Review by SteveYurongSu — APPROVED** [review]
LGTM

**Pull request merged**